src/components/UploadingJobsDisplay/components/FileItem/FileItem.js

```diff
@@ -3,27 +3,30 @@ import React, {
   Fragment,
 } from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import {
+  FormattedMessage,
+  FormattedDate,
+} from 'react-intl';
 import classNames from 'classnames';
+import noop from 'lodash/noop';
```

Comment:
Let's be consistent and use named import.
```suggestion
import { noop } from 'lodash';
```

Reply:
Will do.